src/Compilers/CSharp/Portable/Binder/Semantics/OverloadResolution/OverloadResolution.cs

```diff
@@ -2549,10 +2557,10 @@ internal EffectiveParameters(ImmutableArray<TypeSymbolWithAnnotations> types, Im
                     continue;
                 }
                 var parameter = parameters[parm];
-                types.Add(_binder.GetTypeOrReturnTypeWithAdjustedNullableAnnotations(parameter));
+                types.Add(binder.GetTypeOrReturnTypeWithAdjustedNullableAnnotations(parameter));
```

Comment:
>GetTypeOrReturnTypeWithAdjustedNullableAnnotations [](start = 33, length = 50)

Probably no need to go through this helper given the new approach, i.e. do all nullability related work in flow analysis.

Reply:
Updated.